Destiny.ScrimTracker.Logic/Models/Match.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Destiny.ScrimTracker.Logic.Models
+{
+    public class Match
+    {
+        public IEnumerable<GuardianMatchResult> AlphaTeam { get; set; }
+        public IEnumerable<GuardianMatchResult> BravoTeam { get; set; }
+        public int AlphaTeamScore { get; set; }
+        public int BravoTeamScore { get; set; }
```

Comment:
We should track the match type as well.

Reply:
Also how will this handle the Rumble type where there are no teams?

Reply:
Thinking futher, I think we should introduce a new model `MatchTeam`, and this model should have a list of `MatchTeam` objects associated with it. This way we can handle many configurations of teams.

Reply:
hmm. i think this makes sense. this would cover the rumble situation. you could then also just loop through, or maybe even have a `MatchTeam Winner` property on the Match